products/kuma.md

```diff
@@ -0,0 +1,51 @@
+---
+title: kuma
+category: app
+tags: cncf configuration-management
+permalink: /kuma
+alternate_urls:
+-   /kuma
+releasePolicyLink: https://github.com/kumahq/kuma/blob/master/versions.yml
```

Comment:
```suggestion
```

https://github.com/kumahq/kuma/blob/master/versions.yml is not really a documentation. I think it would be preferable to remove this link and write a small description saying something like this:

> Kuma release and support policy is not clearly documented. But EOL dates are documented [in the project's GitHub repository](https://github.com/kumahq/kuma/blob/master/versions.yml).

WDYT ?

Reply:
I think that sounds reasonable